index.html

```diff
@@ -95,9 +95,15 @@ <h2>Adventure Awaits</h2>
         <div class="col-sm-12">
           <!-- JavaScript Component: Collapse -->
           <button class="btn btn-primary" type="button" data-toggle="collapse" data-target="#collapseExample" aria-expanded="false" aria-controls="collapseExample">
-            <h2>Destinations</h2>
-          </button>    
-          <table id="collapseExample" class="table table-striped collapse"><!-- JavaScript Component: Collapse -->
+            Collapse
+          </button>
```

Comment:
Really great job in implementing this collapse button! A cool fact, there is a timer in the back-end of BS4 that allows for it to look more gradual. Otherwise, when you expand and/or collapse, it looks abit funky.